pattern-library/src/scss/specialised/_site-header.scss

```diff
@@ -21,6 +21,12 @@
   @media print {
     display: none;
   }
+
+  @include mq($upd-site-header-breakpoint-navigation) {
+    display: flex;
```

Comment:
what's the supported browsers for this site?

Reply:
Hey there, I queried this with @alfraser86 .. Chrome, Safari, Firefox current version + 1 version back.